src/main/software/amazon/event/ruler/ByteMachine.java

```diff
@@ -85,7 +86,7 @@ Set<NameState> transitionOn(String valString) {
         boolean fieldValueIsNumeric = false;
         if (hasNumeric.get() > 0) {
             try {
-                final double numerically = Double.parseDouble(valString);
+                final double numerically = parseDouble(valString);
```

Comment:
would prefer to follow `Class.staticMethod(...)` pattern here.

Reply:
ook, I have changed the import